### PR TITLE
vendor, Bump multus-client to v1.3.0

### PR DESCRIPTION
**Pull request by RamLavi**
**What this PR does / why we need it**:
This PR bumps multus-client vendor to v1.3.0, to support latest api.

In order to use the latest network-status annotation api, that among
else exposes pci-device information on sriov secondary nic, bumping
multus-client vendor to v1.3.0

**Which issue(s) this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when PR gets merged)*:
Fixes #

**Special notes for your reviewer**:

**Release note**:

```release-note
NONE
```

### CODE REVIEW

File: go.mod [hunk 1]
```diff
@@ -158,6 +158,7 @@ require (
 
 replace (
 	github.com/golang/glog => ./staging/src/github.com/golang/glog
+	github.com/k8snetworkplumbingwg/network-attachment-definition-client => github.com/k8snetworkplumbingwg/network-attachment-definition-client v1.3.0
```
[the remark this answers is not in the document]
Comment (RamLavi) [reply go.mod]: yep totally. [Done](https://github.com/kubevirt/kubevirt/compare/af5c8a8ed29b77a49a17ed8a055ed9a708cedae9..cab062dab6dfdf5b2d96d755009bb3fc2ec22758)

**Review by orelmisan — COMMENTED** [review]
Thank you for the PR @RamLavi.
Should the change to `pkg/virt-controller/watch/BUILD.bazel` be here?
It is not present in the matching commit of PR #8089

**Comment by RamLavi** [comment]
> Thank you for the PR @RamLavi. Should the change to `pkg/virt-controller/watch/BUILD.bazel` be here? It is not present in the matching commit of PR #8089

not sure, lemme `make generate` to be sure

**Review by orelmisan — APPROVED** [review]
Thank you!

**Review by orelmisan — APPROVED** [review]

**Review by EdDev — APPROVED** [review]
Thanks!

Could you please copy the reasoning from the commit to the PR description?

**Review by ormergi — APPROVED** [review]

**Pull request merged**